apps/condo/domains/ticket/components/IncidentHints.tsx

```diff
@@ -0,0 +1,235 @@
+import {
+    IncidentWhereInput,
+    Incident as IIncident,
+    IncidentClassifierIncidentWhereInput,
+    IncidentStatusType,
+    IncidentClassifier as IIncidentClassifier,
+    IncidentClassifierWhereInput,
+} from '@app/condo/schema'
+import { Col, Row, RowProps } from 'antd'
+import dayjs from 'dayjs'
+import get from 'lodash/get'
+import React, { useCallback, useEffect, useState } from 'react'
+import { IntlShape } from 'react-intl/src/types'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography, Alert } from '@open-condo/ui'
+
+import { shortenText } from '@condo/domains/common/utils/text'
+import {
+    IncidentProperty,
+    Incident,
+    IncidentClassifierIncident,
+} from '@condo/domains/ticket/utils/clientSchema'
+
+
+const INCIDENTS_GUTTER: RowProps['gutter'] = [0, 24]
+const DESCRIPTION_GUTTER: RowProps['gutter'] = [0, 14]
+
+type classifierDataType = Pick<IIncidentClassifier, 'category' | 'problem'>
+
+type IncidentHintsProps = {
+    propertyId: string
+    organizationId: string
+    classifier?: classifierDataType
+    dateISO?: string
+    onlyActual?: boolean
+}
+
+type IncidentHintProps = {
+    incident: IIncident
+}
+
+const formatDate = (intl: IntlShape, date?: string) => {
+    if (!date) return '…'
+    return dayjs(date).format('DD.MM.YYYY HH.mm')
+}
+
+const IncidentHint: React.FC<IncidentHintProps> = (props) => {
+    const intl = useIntl()
+    const MoreLabel = intl.formatMessage({ id: 'incident.hints.more.label' })
+
+    const { incident } = props
+
+    return (
+        <Alert
+            showIcon
+            type='warning'
+            description={
+                <Row gutter={DESCRIPTION_GUTTER}>
+                    <Col span={24}>
+                        <Row>
+                            <Col span={24}>
+                                <Typography.Text strong>
+                                    {shortenText(get(incident, 'details', ''), 100)}
```

Comment:
You can use ellipsis prop for that. This logic already incapsulated by most of antd components

Reply:
It blocked on the typography of the new ui kit

Reply:
Looks like it needs to be fixed)

Reply:
It turns out that the matter is in the `Typography.Text` component - it cannot be cut by the number of lines.
Now I use `Typography.Paragraph` - it allows you to do this